src/models/storage/batch.py

```diff
@@ -200,6 +200,31 @@ def sort(self, by=None):
             by = [self.identifier_column]
         self._frames.sort_values(by=by, ignore_index=True, inplace=True)
 
+    def sort_orderby(self, by=None, sort_type=None):
+        """
+        in_place sort for orderby
+
+        Args:
+            by: list of column names
+            sort_type: list of True/False if ASC for each column name in 'by'
+                i.e [True, False] means [ASC, DESC]
+        """
+        if by is None and self.identifier_column in self._frames:
+            by = [self.identifier_column]
```

Comment:
Should we do this? Atleast let's log a warning message before doing it

Reply:
Removed this section as per comments from the weekly team discussion